app/src/test/java/ch/epfl/qedit/model/ServerTest.java

```diff
@@ -0,0 +1,28 @@
+package ch.epfl.qedit.model;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class ServerTest {
+
+    @Test
+    public void test() {
+        Server server = Server.getInstance();
+        assertTrue(!server.isOut());
+        server.initServer();
+        server.signOut();
+        assertTrue(server.isOut());
+    }
+
+    @Test
+    public void test2() {
+        Server server = Server.getInstance();
+        server.initServer();
+
+        assertFalse(server.createUser("", ""));
```

Comment:
Shouldn't the server be able to create the user ?

Reply:
That's what createUser do. Or at least in this case it sends an email and password. But later we could add more things but we didn't spoke yet about what kind of things we were going to store on the server and how we can access it. 

Firebase let us access the database by email, google, facebook, and some other methods.

Reply:
Then, why is it `assertFalse` and not `assertTrue` ? Creating a new user should succeed, right ?